airbyte-cdk/python/airbyte_cdk/connector.py

```diff
@@ -47,10 +47,14 @@ def configure(self, config: Mapping[str, Any], temp_dir: str) -> TConfig:
         """
 
     @staticmethod
-    def read_config(config_path: str) -> TConfig:
+    def read_json_file(config_path: str) -> TConfig:
         with open(config_path, "r") as file:
             contents = file.read()
-        return json.loads(contents)
+
+        try:
+            return json.loads(contents)
+        except json.JSONDecodeError as error:
+            raise ValueError(f"Could not read json file {config_path}: `{error}`")
```

Comment:
```suggestion
            raise ValueError(f"Could not read json file {config_path}: {error}. Please ensure that it is a valid JSON.")
```

Reply:
I'll take the action item to see in which case JSONDecodeError might get raised. As of now, we have identified:
* File with invalid JSON
* File does not exist
Based on that (and a research to see if there are more cases), I'll update the error message to be more precise. Thanks Sherif